docs/README.erb.md

```diff
@@ -11,23 +11,19 @@ _Laurent/2016-<%=Time.new.year%>_
 
 This gem provides a command line interface to Aspera Applications and a ruby API to Aspera transfers.
 
-Location:
-[https://rubygems.org/gems/aspera](https://rubygems.org/gems/aspera)
+Location (once released):
+[https://rubygems.org/gems/aspera-cli](https://rubygems.org/gems/aspera-cli)
 
 Disclaimers:
 
-* Aspera, FASP are owned by IBM
-* This GEM is not endorsed/supported by IBM/Aspera
-* Use at your risk (not in production environments)
-* This gem is provided as-is, and is not intended to be a complete CLI, or industry-grade product.
-* some features may not be fully validated
-* IBM provides an officially supported Aspera CLI: [http://downloads.asperasoft.com/en/downloads/62](http://downloads.asperasoft.com/en/downloads/62) .
+* This as not yet been officially released so things may change
```

Comment:
same typo here

Reply:
```suggestion
* This has not yet been officially released so things may change
```
Ah ah, silly me, I forgot to change the actual source of the issue...